README.md

```diff
@@ -35,13 +35,12 @@
 
 To use this library you need to ensure you match up with the correct version of React Native you are using.
 
-| `stream-chat-react-native` version  | React Native Version  | `stream-chat` Version |
-| ----------------------------------  | --------------------- | --------------------- |
-| `3.2.0`                             | `>= 0.60`             | `>= 3.5.1`            |
-| `3.0.0`                             | `>= 0.60`             | `>= 3.0.0`            |
-| `2.x.x`                             | `>= 0.60`             | `< 3.0.0`             |
-| `1.x.x`                             | `>= 0.59`             | `< 3.0.0`             |
-| `0.x.x`                             | `*`                   | `< 3.0.0`             |
+| `stream-chat-react-native` | `react-native-reanimated` | `react-native` | `stream-chat` |
+|----------------------------|---------------------------|----------------|---------------|
+| `4.0.0`                    | `>= 2.3.0-beta.2`         | `>= 0.66`      | ``>= ~3.13.1` |
+| `3.8.3`                    | `>= 2.2.1`                | `>= 0.65`      | `>= ~3.13.1`  |
```

Comment:
The `react-native` version might be a bit misleading here, since 3.8.3 or 4.0.0 does support react-native >= 0.60.

Reply:
I think we need another table to note down react-native version against minimum required version of stream-chat-react-native.

Reply:
Also how about we move this table to documentation here - https://getstream.io/chat/docs/sdk/reactnative/#add-dependencies?

Reply:
> The `react-native` version might be a bit misleading here, since 3.8.3 or 4.0.0 does support react-native >= 0.60.

Yeah I was a bit unsure what to do with those, so I tried to see how it was resolved by yarn and use that. But reanimated is the stricter dependency for rn to follow, so could probably show that better

> I think we need another table to note down react-native version against minimum required version of stream-chat-react-native.

Almost seems like three tables make sense 😅 Wanna have a chat tomorrow to brainstorm how best to present this:

Reply:
Note to self -> move to "getting started" in the docs